test/src/test/java/jenkins/security/Security857Test.java

```diff
@@ -102,6 +101,7 @@ private String parseJelly(String jelly) throws Exception {
      * @param escape How the escape-by-default directive was set. null: not set, true: set to true, false: set to false
      */
     private void checkResponse(String response, Boolean escape) {
+        String EVIDENCE = "<script> alert";
```

Comment:
if you're inlining it should update the casing...
```suggestion
        String evidence = "<script> alert";
```